### ISSUE TITLE
Bash version of script

### ISSUE THREAD

**Issue by kakposoe** [jszklarz-haventec/makeitdark#9]
I have used the python script and everything worked fine and smoothly. However, as a dependancy a user of this script will need to install `python` (I already had this so it's not an issue for me).

It would be good to provide both `python` and `bash` versions for those who don't have `python` and have no need for it after the **makeitdark** install.

**Comment by LostConnection** [comment]
Good point, I can write some bash script but it will most likely be a very simple text appending script and will most likely not support windows because of the dynamic installation path.

**Comment by Piees** [comment]
Personally I'd assume packaging it into an executable would have a similar effect of not needing the dependency, however it wouldn't be as agnostic as bash script.
Could however adding packaged versions in a dist folder be helpful?

**Comment by ammmze** [comment]
I've got a bash script for this in my own project I threw together for the devs at my company. https://github.com/MashupMill/slack-themes/blob/master/install.sh

It works relatively similar to your python script using markers to mark out the start and end, so you can strip it out.

You could probably strip out the dev mode stuff. I use that when I'm testing CSS changes. It just spins up a little http server (via nodejs) serving up my local CSS files.
